Guesser/PropelORMFieldGuesser.php

```diff
@@ -73,6 +73,8 @@ public function getManyToMany($model, $fieldPath)
         if ($relation) {
             return \RelationMap::MANY_TO_MANY === $relation->getType();
         }
+
+        return false;
```

Comment:
Method return statements are inconsistent, on line 74 it will return "TRUE", otherwise, method is type of "void". 

Method is declared to return boolean, so this makes method to be consistent with its signature.